tests/modules/nf-scil/reconst/dtimetrics/nextflow.config

```diff
@@ -0,0 +1,21 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
+    withName: "RECONST_DTIMETRICS" {
+        ext.ad = true
+        ext.evecs = true
+        ext.evals = true
+        ext.fa = true
+        ext.ga = true
+        ext.rgb = true
+        ext.md = true
+        ext.mode = true
+        ext.norm = true
+        ext.rd = true
+        ext.tensor = true
+        ext.nonphysical = true
+        ext.pulsation = true
+        ext.residual = true
```

Comment:
I think you should pass false flags for some metrics to your mask test using ``withName: "test_reconst_dtimetrics_with_b0mask:RECONST_DTIMETRICS`` (you could only output the fa for example) to validate that the metrics flag are working correctly.